plugins/hydra_ray_launcher/setup.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from setuptools import find_namespace_packages, setup
+
+with open("README.md", "r") as fh:
+    LONG_DESC = fh.read()
+    setup(
+        name="hydra-ray-launcher",
+        version="0.1.0",
+        author="Jieru Hu",
+        author_email="jieru@fb.com",
+        description="Hydra Ray launcher plugin",
+        long_description=LONG_DESC,
+        long_description_content_type="text/markdown",
+        url="https://github.com/facebookresearch/hydra/",
+        packages=find_namespace_packages(include=["hydra_plugins.*"]),
+        classifiers=[
+            "License :: OSI Approved :: MIT License",
+            "Programming Language :: Python :: 3.7",
```

Comment:
I think it's reasonable to declare this plugin as requiring 3.7 or newer.
If you show me the pickle errors you are getting with 3.6 I might have an idea.

Reply:
I put it  in the description section on top of this PR :) copy pasting here:

cannot run the plugin in python 3.6 (cloudpickle issue, same error message as #428, failed run here https://app.circleci.com/pipelines/github/jieru-hu/hydra/611/workflows/0d99e3b0-1442-446f-857b-f476a7707b6d/jobs/6648 ), I tried a few things (updating cloudpickle version etc but was not able to resolve it.)

Reply:
Is this something you are able to reproduce in a small standalone example?
Are you using the latest version of cloudpickle?